src/pages/about/about.js

```diff
@@ -82,6 +97,80 @@ export default function About() {
           </div>
         </div>
       </section>
+
+      <section className="section-four">
+        <div className="form-background">
+          <div className="form-title">
+            <p id="title-text">Contact us</p>
+          </div>
+          <div >
+          <form onSubmit={handleSubmit}>
+            <div className="first-input-row">
+                <div className="field">
+                    <label className="label">First Name*</label>
+                    <div>
+                      <input
+                        className="small-input"
+                        name="firstName"
+                        value={formdata.firstName}
+                        onChange={handleChange}
+                      />
+                    </div>
+                  </div>
+                  <div className="field">
+                    <label className="label">Last Name*</label>
+                    <div >
+                      <input
+                        className="small-input"
+                        name="lastName"
+                        value={formdata.lastName}
+                        onChange={handleChange}
+                      />
+                    </div>
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Email*</label>
+                <div>
+                  <input
+                    className="large-input"
+                    name="email"
+                    value={formdata.email}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Phone</label>
+                <div >
+                  <input
+                    className="large-input"
+                    name="phone"
+                    value={formdata.phone}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className="field">
+                <label className="label">Message</label>
+                <div >
+                  <textarea
+                    className="message-input"
+                    name="message"
+                    value={formdata.message}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div >
+                <button className="submit-button" type="submit">
+                  Send
+                </button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </section>
```

Comment:
This entire section which creates the contact form should be refactored to its own component.